nixos/modules/system/activation/top-level.nix

```
@@ -306,6 +313,21 @@ in
       '';
     };
 
+    system.includeBuildDependencies = mkOption {
+      type = types.bool;
+      default = false;
+      description = lib.mdDoc ''
+        Whether to include the build closure of the whole system in
+        its runtime closure.  This can be useful for making changes
+        fully offline, as it results in all sources and patches
+        required to build the system being included.
+
+        Note that this increases closure size drastically: a minimal
```

```suggestion
        This increases the system's size drastically: a minimal
```

... or do some people scan for "Note" in text? I don't think I do.

---

(I do, but I am fine either way)

---

```suggestion
        Note that this increases the system size drastically: a minimal
```